libexec/jenv-init

```diff
@@ -47,7 +47,20 @@ abs_dirname() {
   cd "$cwd"
 }
 
-root="$(abs_dirname "$0")/.."
+set +e
+if [ "$( uname -s )" == "Darwin" -a -n "$( which brew )" ]; then
+    jenv_prefix="$( brew --prefix jenv )"
```

Comment:
```suggestion
    jenv_prefix="$(brew --prefix)/opt/jenv"
```

While your invocation is correct, I’d suggest this improvement for performance.

On my 2017 MacBook Pro, this consistently saves over 700 ms:

```
$ time brew --prefix
/usr/local

real	0m0.038s
user	0m0.016s
sys	0m0.017s
```

vs.

```
$ time brew --prefix jenv
/usr/local/opt/jenv

real	0m0.708s
user	0m0.499s
sys	0m0.185s
```

These savings add up when you open several Terminal windows at once, which `Terminal.app` does after e. g. a crash or reboot.

Reply:
Thank for the proposition.
I am working on it.

I work on another way to enable the homebrew support (see https://github.com/jenv/jenv/tree/feature-homebrew )


In order to enable it, you will pass another parameter in the jenv init

`eval (jenv init --homebrew -)`

I also add a way to update the plugin, as the symlink are currently also targeting old location in homebrew after an update ( #232 ).

Reply:
What do you think ? should be ok to add extra parameter to enable homebrew support?

Reply:
Thanks so much @gcuisinier!
Would you also accept a `JENV_INSTALL_DIR` environment variable instead of a `--homebrew` switch?

The thing is, Homebrew can’t automatically modify user profiles. We can control the invocation via the existing `jenv` shim though. (But only for all of `jenv`, not for e. g. `jenv init`.)

That would be great because it would allow Homebrew users to get the fix automatically.
How do you feel about that?

Reply:
> Would you also accept a JENV_INSTALL_DIR environment variable instead of a --homebrew switch?
Not sure to understand the point. 

But if I well understand, you want to have the fit automatically "used" for anybody using homebrew, without having to modify their .bash_profile from

`eval $(jenv init -)` to `eval $(jenv init --homebrew -)` ?

Reply:
@gcuisinier Exactly, yes, that’s the point!
Homebrew never modifies a user profile.

But injecting a variable is fine, and we can do it automatically.

Reply:
Ok, I understand.

But not sure to see what do you mean by "injecting a variables" but without modifiying the user profile.

Today, to install jenv, you have to add this jenv init command. Homebrew itself does not do that.

And if you want to define JENV_INSTALL_DIR before init, you will also need to do it manually no ? And homebrew will not help neither.


So please if you have advice, it will be very welcome :)